pages/docs/reference/kapt.md

```diff
@@ -96,7 +96,7 @@ kapt {
 
 </div>
 
-## Gradle Build Cache Support (since 1.2.20)
+## Gradle build cache support (since 1.2.20)
```

Comment:
I think that "Gradle Build Cache" is kind of a proper noun, so all words should be capitalized

Reply:
In [Gradle docs](https://docs.gradle.org/current/userguide/build_cache.html), it's capitalized only in headings. We use a different heading capitalization style: Gradle uses the traditional `Title Case` and we prefer `sentence case` as does [Microsoft](https://docs.microsoft.com/en-us/style-guide/capitalization), for example.
In sentences, Gradle spells it `build cache` (not a proper noun), so I'd stick to this spelling in both headings and sentences.